### PR TITLE
Add navigation link from individual organization back to all organizations

### LINKED ISSUE #3937: Improve navigation for the All Casa Admin organizations
**What type(s) of user does this feature affect?**

 - all casa admins

**Description**
No current navigation links from the individual organization pages back to the CASA Organizations list.

**Screenshots of current behavior, if any**  
You can paste images on the clipboard here

_QA Login Details:_  
[Link to QA site](https://casa-qa.herokuapp.com/)  

Login Emails: 
- volunteer1@example.com  view site as a volunteer
- supervisor1@example.com view site as a supervisor
- casa_admin1@example.com view site as an admin

password for all users: 12345678  

### Questions? Join Slack!

We highly recommend that you join us in slack https://rubyforgood.herokuapp.com/ #casa channel to ask questions quickly and hear about office hours (currently Tuesday 6-8pm Pacific), stakeholder news, and upcoming new issues.

### PR DESCRIPTION
**Pull request by elhalvers**
### What github issue is this PR for, if any?
Resolves #3937

### What changed, and why?
Add navigation link from individual organization back to all organizations for improved UX.

Refactored existing spec.

### How will this affect user permissions?
NA

### How is this tested? (please write tests!) 💖💪
Wrote a system spec test.

### Screenshots please :)

Screen Shot 2022-09-13 at 10 32 48 PM

### Feelings gif (optional)
What gif best describes your feeling workinghttps://giphy.com/ on this issue? https://giphy.com/

`"sliding in to home"`

How to embed:
`alt text`

### Feedback please? (optional)
We are very interested in your feedback! Please give us some :) https://forms.gle/1D5ACNgTs2u9gSdh9

### CODE REVIEW

**Comment by elhalvers** [comment]
Pushed 2 new commits for ERB lint issue and standardrb lint issue. Not yet sure I understand the rspec issue.

Added screenshot to PR for proposed change. Wasn't successful adding the embedded feeling gif ; ) 

Thank you! : )

File: Gemfile.lock [hunk 1]
```diff
@@ -1,71 +1,71 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (7.0.3.1)
-      actionpack (= 7.0.3.1)
-      activesupport (= 7.0.3.1)
+    actioncable (7.0.4)
```
Comment (compwron) [inline Gemfile.lock:4]: interesting upgrade but seems fine

**Review by compwron — APPROVED** [review]
nice! yay tests :)

File: Gemfile.lock (hunk 1 above)
Comment (elhalvers) [reply Gemfile.lock:4]: Thank you! : )

**Pull request merged**